lib/DDG/Spice/LeakDB.pm

```diff
@@ -22,7 +22,7 @@ triggers startend => 'hashme', 'leakdb';
 spice to => 'http://api.leakdb.abusix.com/?j=$1';
 spice wrap_jsonp_callback => 1;
 
-handle remainder_lc => sub{
+handle remainder => sub{
```

Comment:
I think you'll want to keep `remainder_lc`, otherwise the remainder won't be lowercased when it's passed into your regex below, making your IA case sensitive.